### ISSUE TITLE
Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2

### ISSUE THREAD

**Issue by Bryceless** [deeppomf/DeepCreamPy#93]
**Describe the bug**
I get this bug when running DeepCreamPy on an Intel Core i5-7600k:

Using TensorFlow backend.
T:\src\github\tensorflow\tensorflow\core\platform\cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2

I have tried using the decensor.exe and the batch that decensors mosaics.

I installed DeepCreamPy using the binaries provided.

**Desktop (please complete the following information):**
 - OS: Windows 10 x64
 - Version 17763.194
 - It supports AVX2 according to the Intel CPU page

![image](https://user-images.githubusercontent.com/5755974/49905646-8b198b00-fe3c-11e8-8cb8-d520648b3842.png)

**Comment by ToxicFrazzles** [comment]
If the decensoring worked then that message does not indicate a bug. Your CPU just has extra functionality that isn't being used.

**Comment by Bryceless** [comment]
> If the decensoring worked then that message does not indicate a bug and can be disregarded. Your CPU just has extra functionality that isn't being used.

It doesn't output any images in general.

Nothing is inside of the decensor_output folder after this window closes out.

**Comment by ToxicFrazzles** [comment]
Could you try running the exe from either the command line or PowerShell and post the full output?

**Comment by Bryceless** [comment]
Z:\Users\Bryce\Downloads\DeepCreamPy_1.3.0-beta_win64>decensor.exe
Using TensorFlow backend.
2018-12-13 14:40:34.022086: I T:\src\github\tensorflow\tensorflow\core\platform\cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
--------------------------------------------------------------------------

Z:\Users\Bryce\Downloads\DeepCreamPy_1.3.0-beta_win64>

**Comment by ToxicFrazzles** [comment]
did you place a censored image in the "decensor_input" folder?

**Comment by Bryceless** [comment]
Yeah, I have the censored image with the green overlay on the censored regions in the decensor_input folder, while the censored image without it in decensor_input_original.

**Comment by ToxicFrazzles** [comment]
I've nearly run out of ideas here, but is the image in png format?

**Comment by Bryceless** [comment]
The image is in .JPG format.

I can try converting the image to a .PNG if that is required.

**Comment by Bryceless** [comment]
Yeah, that fixed the issue. Thanks!

**Comment by Flyersfan976** [comment]
I have this message
![Capture](https://user-images.githubusercontent.com/50161270/57005010-2324d080-6ba2-11e9-8748-055255ba7278.JPG)

**Comment by Curtistic** [comment]
The issue has appeared in the latest release 2.0.0: 

The error message is approximately: 
Using TensorFlow backend.
\cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX

I tried again with the mosaic but the same thing happened. I tried again and let it run instead of closing the command box, but my laptop crashed and I had to restart.

Windows 10 64x, latest update
